### PR TITLE
Feature/start cypress

### PR DESCRIPTION
**Pull request by Wperricone**
## What does this PR actually DO? / Issue you solved
- This PR adds preliminary Cypress Tests to the pages we have components for so far because we did not have any Cypress tests for our code yet. 

## Where are the Changes:
-addItemFormView.cy.js, itemCardView.cy.js, itemExpandedview.cy.js, landingPageView.cy.js, userProfileView.cy.js were all created. Some changes were made in the corresponding JS files of Form.js, ItemCard.js, ItemExpanded.js, LandingPage.js, UserProfile.js

## How was this tested:
- For the components we have rendered, they were tested through Cypress. For the components we don't yet have rendered, we will test them via Cypress once we merge 

## What are the relevant tickets:
- Issues 24, 25, 26 and 30

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -6,4 +6,6 @@ workflows:
     jobs:
       # "cypress" is the name of the imported orb
       # "run" is the name of the job defined in Cypress orb 
-      - cypress/run
\ No newline at end of file
+      - cypress/run:
+          start: npm start
+          wait-on: 'http://localhost:3000'
```
Comment (JoeRupp) [inline .circleci/config.yml:11]: Nice! Glad we solved this!

File: cypress/e2e/1-getting-started/landingPage.cy.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
+describe('Craft Circle landing page view', () => {
+  beforeEach(() => {
+    //cy.intercept("craft-circle-be.herokuapp.com/graphql", {fixture: "userProfile.json"})
+    cy.visit("http://localhost:3000")
+  });
+
```
Comment (JoeRupp) [inline cypress/e2e/1-getting-started/landingPage.cy.js:6]: Nice work on the intro tests!

File: cypress/e2e/1-getting-started/addItemFormView.cy.js [hunk 3]
```diff
@@ -0,0 +1,57 @@
+describe('Craft Circle landing page view', () => {
+  beforeEach(() => {
+    // cy.intercept("craft-circle-be.herokuapp.com/graphql", {fixture: "userProfile.json"})
+    cy.visit("http://localhost:3000")
+  });
+
```
Comment (stephanie-roe) [inline cypress/e2e/1-getting-started/addItemFormView.cy.js:6]: Thank you for setting up our testing structure! I know we will add to cover remaining user flows once more functionalities are plugged in.

**Review by stephanie-roe — APPROVED** [review]
Thank you for taking an iterative approach to solving our circle ci issues! Our testing structure is looking solid.

**Pull request merged**